app/graph/types/account_source_type.rb

```diff
@@ -0,0 +1,29 @@
+AccountSourceType = GraphqlCrudOperations.define_default_type do
+  name 'AccountSource'
+  description 'AccountSource type'
+
+  interfaces [NodeIdentification.interface]
+
+  field :id, field: GraphQL::Relay::GlobalIdField.new('AccountSource')
+  field :account_id, types.Int
+  field :source_id, types.Int
+  field :permissions, types.String
```

Comment:
All default types have `permissions` (as per https://github.com/meedan/check-api/blob/develop/app/graph/mutations/graphql_crud_operations.rb#L132), so you can hide this